scalableminds/webknossos
# Remove unlink fallback layer functionality

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a hybrid annotation with a fallback segmentation layer and check the layer settings. The UI should contain no traces of the "Unlink Fallback Layer" functionality and the remaining buttons should be rendered correctly
- CI linting should catch dead code 

### Issues:
- fixes #6137 

------
(Please delete unneeded items, merge only when none are left open)
